src/components/Header/components/LocationSearch/LocationSearch.jsx

```diff
@@ -1,14 +1,65 @@
 import classes from './LocationSearch.module.scss';
+import React from 'react';
+import SearchDropDown from './components/SearchDropDown/SearchDropDown';
 
-function LocationSearch() {
-  return (
-    <div className={classes.currentLocation}>
-      <span className={classes.title}>current city:</span>
-      <a className={classes.value} href="#">
-        Gomel
-      </a>
-    </div>
-  );
+class LocationSearch extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentLocation: 'Gomel',
+      isOpenDropDown: false
+    };
+  }
+
+  dropdownContainer = React.createRef();
+
+  componentDidMount() {
+    document.addEventListener('mousedown', this.handleClickOutsideDropdown);
```

Comment:
would this event fire if you click right above your dropdown?

Reply:
No, but already I have fixed it. Problem was in wrappers.